ajax/libs/color-scheme/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "color-scheme",
+  "filename": "color-scheme.min.js",
+  "version": "1.0.0",
+  "description": "Generate pleasant color schemes",
+  "main": "lib/color-scheme.js",
+  "scripts": {
+    "test": "mocha --compilers coffee:coffee-script/register",
+    "prepublish": "cake build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/c0bra/color-scheme-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/c0bra/color-scheme-js/issues"
+  },
+  "keywords": [
+    "color",
+    "scheme",
+    "generator",
+    "random",
+    "picker",
+    "web"
+  ],
+  "npmName": "color-scheme",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```

Comment:
Change the file rule to `"color-scheme*"`. We wouldn't want any files which don't need to be hosted in case something comes up in future releases.